src/controllers/core/controller.js

```diff
@@ -102,16 +102,14 @@ class Controller
     }
 
     /**
-     * Updates the current amount of money. Note that amount is added to GAME.model.config.money,
-     * so input negative amount to subtract!
-     *
-     * @param {float} amount - the amount to be added to the current amount of money
-     *
-     * @protected
-     */
-    _updateMoney(amount)
-    {
-        GAME.model.config.money = GAME.model.config.money + parseFloat(amount);
-        $("#money").html(GAME.model.config.money.toFixed(2));
-    }
-}
+         * Updates the current amount of money. Note that amount is added to GAME.model.config.money,
+         * so input negative amount to subtract!
+         *
+         * @param {float} amount - the amount to be added to the current amount of money
+         *
+         * @protected
+         */
+        _updateMoney(amount)
+        {
+            GAME.model.config.money = GAME.model.config.money + parseFloat(amount);
+            $("#money").html(GAME.model.config.money.toFixed(2));
```

Comment:
This is not correct. Could you carefully look at what it was like originally?

Else just copy this file https://raw.githubusercontent.com/LCfP/Serious-Game-for-Kids/master/src/controllers/core/controller.js and overwrite your entire local controller.js with it.